src/screens/CaptureVehicleSelection/index.js

```diff
@@ -0,0 +1,164 @@
+import { useMonitoring } from '@monkvision/corejs';
+import { useInterval } from '@monkvision/toolkit';
+import { Container } from '@monkvision/ui';
+import { useFocusEffect, useNavigation, useRoute } from '@react-navigation/native';
+import { LinearGradient } from 'expo-linear-gradient';
+import useSnackbar from 'hooks/useSnackbar';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Text, View, useWindowDimensions } from 'react-native';
+import { Card, List, useTheme } from 'react-native-paper';
+import { useMediaQuery } from 'react-responsive';
+
+import { version } from '@package/json';
+import * as names from 'screens/names';
+import Artwork from '../Landing/Artwork';
+import LanguageSwitch from '../Landing/LanguageSwitch';
+import useGetInspection from '../Landing/useGetInspection';
+import { Clients, Workflows, useClient } from '../../contexts';
+import VehicleType from '../Landing/VehicleType';
+import useUpdateInspectionVehicle from '../Landing/useUpdateInspectionVehicle';
+import useZlibCompression from '../Landing/useZlibCompression';
+import styles from './styles';
+
+const ClientParamMap = {
```

Comment:
The `screens/landing` file also has `ClientParamMap` object. Please create a new file `screens/paramsMap.js` for having `ClientParamMap` and `VehicleTypeMap` definitions. This will serve as a common file to export these objects and use them in both `landing` page and this page.